scipy/stats/_continuous_distns.py

```diff
@@ -5670,6 +5680,10 @@ def _stats(self, nu):
         g2 /= nu*mu2**2.0
         return mu, mu2, g1, g2
 
+    def _rvs(self, nu, size=None, random_state=None):
```

Comment:
Since it's private, I don't think it makes sense to give defaults values here. At leat for `random_state` it reads broken otherwize.
```suggestion
    def _rvs(self, nu, size, random_state):
```

Reply:
I agree, I kept it as it is for now since it is used like that in all _rvs methods. It can be fixed in a follow-up PR for all distributions.